int64 flight_state
---
int64 changed
